app/lib/donations/controllers/DonorsChooseDonationController.js

```diff
@@ -433,6 +433,63 @@ function getDonationCount(member) {
 }
 
 /**
+ * Queries Gambit Jr. API to sync donorschoose_bots config documents.
+ *
+ * @param {object} req Express request
+ * @param {object} res Express response
+ */
+DonorsChooseDonationController.prototype.syncBotConfig = function(req, res) {
+  var self = this;
+  if (!req.body.apiKey || req.body.apiKey !== process.env.GAMBIT_API_KEY) {
```

Comment:
Should change this to `api_key` to keep consistent with other params we pass to Gambit API